### ISSUE TITLE
CraftTweaker Integration broken with v.1.0.9

### ISSUE THREAD

**Issue by Fireztonez** [CyclopsMC/IntegratedDynamics#709]
#### Issue type:
- :bug: Bug

____

#### Short description:
When I create recipes with Crafttweaker for Itegrated Dynamics (in that case Squeezer and Mechanical Squeezer) I get an error message No such member of integrateddynamics. My script work fine in version 1.0.8, but with the 1.0.9 the scrips is now broken... So, unless you have change the syntax without updating the Crafttweaker Wiki, I think the Crafttweaker Integration is completly broken. (PS: I look to make sure, and the CT integration is still enabled in the option `integrateddynamicscompat.cfg`)

#### Steps to reproduce the problem:
1. Create a simple Crafttweaker script like:
```mods.integrateddynamics.MechanicalSqueezer.addRecipe(<minecraft:stone>, <minecraft:cobblestone>*2, null, 10);
Squeezer.addRecipe(<minecraft:stone>, <minecraft:cobblestone>, 1.0, <minecraft:cobblestone>, 0.75);
```
2. Start the pack and look the crafttweaker.log
I get an error... See log file...

#### Expected behaviour:
Should work like it work with the version 1.0.8...

____

#### Versions:

- This mod: 1.0.8
- Minecraft: 1.12.2
- Forge: 14.23.5.2823
- CraftTweaker2 : 4.1.16

#### Craftweaker Log file + script file:
https://gist.github.com/Fireztonez/c2d115f8979cddec7eb0b90f8e38c66c

**Comment by rubensworks** [comment]
Possibly caused by the recent CurseForge upload issue, which required me to jump through some mod building hoops

**Comment by rubensworks** [comment]
Will be fixed in 1.0.10
